maltd-frontend/src/components/page/MainPage/MainPage.js

```diff
@@ -117,7 +117,7 @@ export default function MainPage() {
           setDisabledButton(true);
           setDisabledInput(true);
 
-          fetch(`${baseUrl}/api/users/${value}`)
+          fetch(`https://localhost:5001/api/users/${value}`)
```

Comment:
no this needs to stay as `baseUrl`, cant be hard coded to `localhost` with a port

Reply:
Yeah, I accepted the wrong merge change!  Fixed and pushed.

Reply:
Thank you, fixed!